### ISSUE TITLE
Restore form content when reopening / navigating back

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#1311]
A common complaint about history navigation is that it loses the form / HTML input content.

We could automatically save the content of all history entries and add a command to let the user re-insert it on demand.

It's probably better to not re-insert content automatically, since it may contain sensitive data.

**Comment by aadcg** [comment]
I don't think this is an issue that Nyxt should solve. It's the responsibility of certain pages to account for it. Obviously, the renderer couldn't do it by default since it raises many security concerns.
